sphinxcontrib/datatemplates/directive.py

```diff
@@ -16,6 +16,25 @@
 from sphinxcontrib.datatemplates import helpers
 
 LOG = logging.getLogger(__name__)
+_default_templates = None
+
+
+def _templates(builder):
+    global _default_templates
+
+    # Some builders have no templates manager at all, and some
+    # have the attribute set to None.
+    templates = getattr(builder, 'templates', None)
+
+    if not templates:
+        if not _default_templates:
+            from sphinx.jinja2glue import BuiltinTemplateLoader
```

Comment:
I generally like to have all of the imports at the top of the file, unless doing that is going to introduce some sort of circular import case. It's not a big deal, and you could make the change by pushing another patch to this PR or in a separate PR, if you prefer.

Reply:
@dhellmann It is an artifact from the initial approach that I abandoned (where a separate module was used to initialize a template manager and it was imported only if needed).

I'll clean this up a bit and push the changes here.

Reply:
Looks good, thanks!